ecom_test/src/generic_helpers/generic_product_review_helper.py

```diff
@@ -0,0 +1,64 @@
+
+from ecom_test.src.utilities.generic_utilities import generate_random_string, generate_random_email_and_password
+from ecom_test.src.utilities.wooAPIUtility import WooAPIUtility
+
+
+class GenericProductReviewsHelper:
+    """
+    Helper class for managing product reviews in an e-commerce system.
+    """
+
+    def __init__(self):
+        # Initialize the WooAPIUtility for making API calls
+        self.woo_api_utility = WooAPIUtility()
+
+    def create_product_review(self, product_id, review, reviewer, reviewer_email, rating, expected_status_code=201, **kwargs):
+        """
+         Create a product review (Payload) for a specific product.
+
+        :param product_id: The ID of the product to review.
+        :param review: The text content of the review.
+        :param reviewer: The name of the reviewer.
+        :param reviewer_email: The email address of the reviewer.
+        :param rating: The rating given to the product (integer value).
+        :param expected_status_code: The expected HTTP status code (default is 201).
+        :param kwargs: Additional keyword arguments to include in the payload.
+        :return: The response from the API call.
+        """
+        payload = {
+            "product_id": product_id,
+            "review": review,
+            "reviewer": reviewer,
+            "reviewer_email": reviewer_email,
+            "rating": rating
+        }
+
+        payload.update(kwargs)
+
+        return self.woo_api_utility.post('products/reviews', params=payload, expected_status_code=expected_status_code)
+
+    def get_product_review(self, review_id, expected_status_code=200):
+        """
+         Get a specific product review by its ID.
+
+        :param review_id: The ID of the review.
+        :param expected_status_code: The expected HTTP status code (default is 200).
+        :return: The response from the API call.
+        """
+        return self.woo_api_utility.get(f'products/reviews/{review_id}', expected_status_code=expected_status_code)
+
+    def create_random_review_for_product(self, product_id):
```

Comment:
It would be nice if this function would take the 'rating' as argument. You definitely will need to create review other than ratting=5 eventually. so you can add the parameter with default value of 5 if you want. (self, product_id, rating=5

Reply:
done